pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -302,6 +309,13 @@ func (l *Launcher) GetDomain(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.D
 	return response, nil
 }
 
+func getInterfaceStatuses(l *Launcher, domain *api.Domain) []api.InterfaceStatus {
+	interfaceStatuses := l.domainManager.GetInterfaceStatus()
+	interfaceStatuses = agentpoller.MergeAgentStatusesWithDomainData(domain.Spec.Devices.Interfaces, interfaceStatuses)
+
+	return interfaceStatuses
+}
```

Comment:
How about doing this in the `LibvirtDomainManager` under `GetInterfaceStatus`?
Then we do not need any new dependencies in this package.